backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQTest.java

```diff
@@ -52,9 +55,40 @@
 import com.rabbitmq.client.Connection;
 import com.rabbitmq.client.ConnectionFactory;
 
-@ExtendWith(RabbitMQExtension.class)
 class RabbitMQTest {
 
+    private static class RabbitMQTestExtension extends RabbitMQExtension {
```

Comment:
RabbitMQTestExtension is a strange name, because an Extension is already for test.

Maybe you mean RabbitMQNestedTestExtension?